### PR TITLE
More updates

### PR DESCRIPTION
**Pull request by JamesEarle**
Do not merge until we also have the updated package included for zAuction SDK, pending [this PR](https://github.com/zer0-os/zAuction-SDK/pull/65)

### CODE REVIEW

File: test/actions.test.ts [hunk 1]
```diff
@@ -68,184 +59,128 @@ describe("Test Custom SDK Logic", () => {
   };
 
   describe("E2E with Rinkeby", () => {
```
Comment (deep-quality-dev) [inline test/actions.test.ts:61]: It should be `Goerli` :)

Comment (JamesEarle) [reply test/actions.test.ts:61]: lol true, was too focused on the tests themselves. Thanks

File: test/actions.test.ts [hunk 2]
```diff
@@ -67,185 +58,129 @@ describe("Test Custom SDK Logic", () => {
...
-    });
-
-    it("does not flag acceptable content", async () => {
-      var sample = `2Chains`;
-      const sdkInstance = zNSSDK.createInstance(config);
-      const moderation = await sdkInstance.utility.checkContentModeration(
-        sample
-      );
-      expect(moderation.flagged).to.be.false;
+  describe("E2E with Goerli", () => {
```
Comment (deep-quality-dev) [inline test/actions.test.ts:61]: I assume I should have more test cases to test for `getDomainById`, `getDomainsByName`, `getAllDomains`, etc ... @JamesEarle

File: .mocharc.js [hunk 3]
```diff
@@ -3,6 +3,6 @@
 module.exports = {
   exit: true,
   require: "ts-node/register",
-  spec: ["test/**/*.test.ts"],
+  spec: ["test/**/e2e.test.ts"],
```
Comment (Remscar) [inline .mocharc.js:6]: can we leave this as *

Comment (JamesEarle) [reply .mocharc.js:6]: yes, mistakenly left in

**Review by Will-Hyde — APPROVED** [review]

**Pull request merged**